templates/appliance/community.html

```diff
@@ -1,216 +1,213 @@
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
 {% block title %}Ubuntu Appliance Community - Governance and Policies{% endblock %}
 
-{% block meta_description %}Join the Ubuntu community by developing, discussing or even simply downloading one of the Ubuntu Appliances. Anyone can create an Ubuntu Appliance. Have your project benefit from all of the goodness that goes into Ubuntu.{% endblock %}
+{% block meta_description %}
+  Join the Ubuntu community by developing, discussing or even simply downloading one of the Ubuntu appliances. Anyone can create an Ubuntu appliance. Have your project benefit from all of the goodness that goes into Ubuntu.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit
+{% endblock meta_copydoc %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit{% endblock meta_copydoc %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h1>
-        Welcome to our Ubuntu Appliance community
-      </h1>
-      <p>
-        Whether you are a power user, a developer, a member of the Ubuntu project or a new contributor to Ubuntu Appliances, we invite you to join our Ubuntu Appliance Discourse and make your voice heard.
-      </p>
+
+  {% call(slot) vf_hero(
+    title_text='Welcome to our<br /> Ubuntu appliance community',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
       <p>
-        <a href="https://discourse.ubuntu.com/c/appliance">Join the Ubuntu Appliance Discourse</a>
+        Whether you are a power user, a developer, a member of the Ubuntu project or a new contributor to Ubuntu appliances, we invite you to join our Ubuntu appliance Discourse and make your voice heard.
       </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="https://discourse.ubuntu.com/c/appliance">Join our Ubuntu appliance Discourse&nbsp;&rsaquo;</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--3-2 is-cover">
+        {{ image(url="https://assets.ubuntu.com/v1/da070994-hero image.png",
+                alt="",
+                width="1800",
+                height="1128",
+                hi_def=True,
+                loading="auto",
+                attrs={"class": "p-image-container__image"},) | safe
+        }}
+      </div>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <hr class="is-fixed-width p-rule" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          New Ubuntu appliance ideas
+          <br />
+          and proposals
+        </h2>
+      </div>
+      <div class="col">
+        <p>To suggest a new Ubuntu appliance:</p>
+        <hr class="p-rule--muted" />
```

Comment:
To match the spacing on the list dividers 
```suggestion
        <hr class="p-rule--muted u-no-margin--bottom" />
```